### PR TITLE
test(express-instrumentation): add test-all-versions

### PR DESCRIPTION
**Pull request by matteoga**
## Which problem is this PR solving?
- Adds test-all-versions for `express` running tests on the previous minor version `4.17.1` solving https://github.com/open-telemetry/opentelemetry-js-contrib/issues/804

### CODE REVIEW

**Comment by Rauno56** [comment]
Nice! :tada: How would you feel about broadening the tested version range a bit, @matteoga? To include the current, future versions and a couple more of the old, but still used ones.

**Comment by matteoga** [comment]
sounds good, what do you suggest, adding `latest` versions tag and including also 
```
┌─────────┬──────────┬──────────────┬──────────────┬───────────┬──────────┐
│ (index) │ version  │     time     │     tags     │ downloads │ ratio(%) │
├─────────┼──────────┼──────────────┼──────────────┼───────────┼──────────┤
│    2    │ '4.16.4' │ '2018-10-11' │      []      │  471561   │   3.3    │
│    3    │ '4.16.3' │ '2018-03-12' │      []      │  198109   │   1.4    |
```
?

**Comment by Rauno56** [comment]
Yes, possibly add `4.16.2` as well, because the number of downloads for that is more than for some of the other packages we support in total.
However, afaik, TAV doesn't support tags so `^4.16.2` would work well.

File: plugins/node/opentelemetry-instrumentation-express/.tav.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+express:
+  - versions: "^4.17.2"
+    commands: npm run test
+
+  - versions: "4.17.1"
```
Comment (Rauno56) [inline plugins/node/opentelemetry-instrumentation-express/.tav.yml:5]: Any reason you didn't use `^4.16.2` for the range which matches all the listed versions?

File: plugins/node/opentelemetry-instrumentation-express/.tav.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    commands: npm run test
+
+  - versions: "4.16.4"
+    commands: npm run test
+
+  - versions: "4.16.3"
+    commands: npm run test
+
+  - versions: "4.16.2"
+    commands: npm run test
```
Comment (Rauno56) [inline plugins/node/opentelemetry-instrumentation-express/.tav.yml:15]: nit: newline

**Review by Rauno56 — APPROVED** [review]
Thanks!

File: plugins/node/opentelemetry-instrumentation-express/.tav.yml (hunk 1 above)
Comment (blumamir) [reply plugins/node/opentelemetry-instrumentation-express/.tav.yml:5]: I too think it's more appropriate here

**Comment by blumamir** [comment]
style: use a single semantic version range to test relevant versions

**Review by blumamir — APPROVED** [review]

**Review by vmarchaud — APPROVED** [review]

**Pull request merged**